src/python/WMCore/Storage/Plugins/CPImpl.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+"""
+_CPImpl_
+
+Implementation of StageOutImpl interface for plain cp
+
+"""
+import os
+import errno
+import shutil
+
+from WMCore.Storage.StageOutImplV2 import StageOutImplV2
+from WMCore.Storage.StageOutError import StageOutFailure
+
+
+class CPImpl(StageOutImplV2):
+    """
+    _CPImpl_
+
+    Implement interface for plain cp command
+
+    """
+
+    def createOutputDirectory(self, targetPFN):
+        targetdir = os.path.dirname(targetPFN)
+        if not os.path.isdir(targetdir):
+            try:
+                os.makedirs(targetdir)
+            except OSError as exc:
+                if exc.errno != errno.EEXIST:
+                    raise
+        return
+
+    def doTransfer(self, fromPfn, toPfn, stageOut, pnn, command, options, protocol, checksum ):
+        self.createOutputDirectory(toPfn)
+        shutil.copy(fromPfn, toPfn)
```

Comment:
This `shutil.copy()` doesn't copy all the file metadata. But looking at the history, that's how this plugin used to be, so I think we can leave it like that.